utils/colors-vars-map.ts

```diff
@@ -0,0 +1,264 @@
+export const colorsMap = [
```

Comment:
are you sure this entire file is needed?

Reply:
Removed unrelevant parts